### DISCUSSION TITLE
Understanding in-/out-degree popularity correctly

### DISCUSSION THREAD

**Discussion by zhiyzuo** [stocnet/rsiena#14] [Q&A] [answered]
First of all, I really appreciate your efforts in building this package! It is amazing and helpful to do social network research.

I have a hard time understanding the definition of __indegree popularity__ (or __outdegree__). It is defined as
<a href="https://www.codecogs.com/eqnedit.php?latex=inPop&space;=&space;\sum_j&space;x_{ij}&space;x_{&plus;j}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?inPop&space;=&space;\sum_j&space;x_{ij}&space;x_{&plus;j}" title="inPop = \sum_j x_{ij} x_{+j}" /></a>, 
which is the sum of indegree of the alters of an ego.

The positive coefficient of `inPop` is interpreted as tending to attract more ties to the ego (focal node `i`). I wonder why this is the case? I feel like the indegree of the node itself would be a more direct nodal effect on this interpretation. I am confused. May I ask for some guidance over this? Thank you.

**Comment by TomSnijders** [comment, marked as answer]
Dear Zhiya Zuo,

For this type of question we have an email discussion list, which just yesterday was transferred to https://groups.io/g/RSiena.
May I suggest you to register in this list and then to pose the question there?

Best regards,
Tom

=================================================
Tom A.B. Snijders
Professor of Statistics and Methodology
Department of Sociology, University of Groningen
Emeritus Fellow, Nuffield College, University of Oxford
Associate Member, Dept. of Statistics, University of Oxford

**Comment by zhiyzuo** [comment]
Thank you for the information Tom. I've just composed a new topic in the email list and close this comment.
